build.gradle

```diff
@@ -91,8 +91,45 @@ ossIndexAudit {
     colorEnabled = true // if true prints vulnerability description in color. By default is true.
     printBanner = true // if true will print ASCII text banner. By default is true.
 
-    // ossIndexAudit can be configured to exclude vulnerabilities from matching
-    // excludeVulnerabilityIds = ['39d74cc8-457a-4e57-89ef-a258420138c5'] // list containing ids of vulnerabilities to be ignored
+    // Vulnerabilities that can safely be ignored
+    excludeVulnerabilityIds = [
+            // pkg:maven/org.apache.zookeeper/zookeeper@3.4.8 (from flink-test-utils)
+            '1775f19b-5e9c-48f8-8c8f-ef245350531b', '43b8cbe5-324d-416b-b613-3dcaba1b5a6c', 'bec057e0-9945-49c4-92c9-4be669bb5331', '1e65ed27-7a06-4464-a115-c8421c879281',
+
+            // pkg:maven/io.netty/netty@3.7.0.Final (from flink-test-utils)
+            '20be5124-16a3-4d77-8668-d83f04a67808', '20167979-f872-4765-85ef-9b7be870cecb', 'f2a31abe-1af6-4f6f-aeeb-60e0e0d3e981', '6ff63cbd-c4ae-4268-be95-43322746b6be',
+            '8b7d8928-61ee-4708-bf37-feece927a872', 'cbde3175-9c07-491c-836d-2a146b61e3b6', '846fbf13-a0b9-4cab-b820-8415a30326bd', 'b3c3a56f-37c0-4706-bf54-2f61c5c9786f',
+            'a3df5795-2bdb-4ec6-b9c2-babe9ec470e5', '20b79835-7a43-4bea-a985-5e12df1d0b0a',
+
+            // pkg:maven/com.google.guava/guava@16.0.1 (from flink-test-utils)
+            '24585a7f-eb6b-4d8d-a2a9-a6f16cc7c1d0', '8e973be2-4220-410d-a4cb-2de7a755bdbe',
+
+            // pkg:maven/junit/junit@4.12 (from flink-test-utils)
+            '7ea56ad4-8a8b-4e51-8ed9-5aad83d8efb1',
+
+            // pkg:maven/org.jsoup/jsoup@1.10.2 (from com.vladsch.flexmark:flexmark-all)
+            '5dbdb043-212c-4971-9653-d04e1cfc5080',
+
+            // pkg:maven/org.apache.pdfbox/pdfbox@2.0.8 (from com.vladsch.flexmark:flexmark-all)
+            '32bccbbb-bd3f-44f3-a530-5827a93d74db', '2418a058-dfb7-4303-b1d1-ac5262d9e499', '1be5c4d8-4994-4dd6-92b1-ca53ac7e6a17', 'f984e75c-06bf-43dc-86db-3f8dfe82f430',
+            'bd0f0dd9-1356-4dec-a8a3-edc777cc92d5', 'b70e447a-06a1-48b0-a02b-cd5b78862777',
+
+            // pkg:maven/org.apache.pdfbox/xmpbox@2.0.8 (from com.vladsch.flexmark:flexmark-all)
+            '32bccbbb-bd3f-44f3-a530-5827a93d74db', '2418a058-dfb7-4303-b1d1-ac5262d9e499', '1be5c4d8-4994-4dd6-92b1-ca53ac7e6a17', 'f984e75c-06bf-43dc-86db-3f8dfe82f430',
+            'bd0f0dd9-1356-4dec-a8a3-edc777cc92d5', 'b70e447a-06a1-48b0-a02b-cd5b78862777',
+
+            //pkg:maven/org.apache.commons/commons-compress@1.18 (from org.apache.flink:flink-scala_2.12:1.10.3)
+            //NOTE: these vulnerabilities cannot be eliminated UNTIL we upgrade our Flink version to Flink 1.14.0
+            //Need to get Security Exception until that happens
+            '68232267-bb25-4b04-8dec-caf7c11c7293', '69b8043a-3002-48fa-9762-8f6040d83de1', '4102317d-8250-465e-a46d-179d42792b14',
+            '7a6a9dd2-67de-4e2a-b406-7aa4a4ce29cc', '8ea14e38-e6cc-48d9-bfe4-ec89f93596e7',
```

Comment:
So we need to ask about these. To do that, we'll need a good understanding of what the vulnerability is and the implications for the HRI. Can you put that in writing somewhere?

Reply:
> So we need to ask about these. To do that, we'll need a good understanding of what the vulnerability is and the implications for the HRI. Can you put that in writing somewhere?

I dug a little into these vulnerabilities. 
1. The commons-compress lib is used by `flink/flink-core`. 
2. For 3 of these vulnerabilities, this issue is that:

> When reading a specially crafted [ZIP|7Z|TAR] archive, Compress can be made to allocate large amounts of memory that finally leads to an out of memory error even for very small inputs.

3. For one other vulnerability: 

> When reading a specially crafted 7Z archive, the construction of the list of codecs that decompress an entry can result in an infinite loop

4. For the final vulnerability: 

> The file name encoding algorithm used internally in Apache Commons Compress 1.15 to 1.18 can get into an infinite loop when faced with specially crafted inputs.

So, I believe as we discussed in Stand-up a couple of days ago, because our Flink Cluster has access restricted to it, this issue can only be precipitated by an Internal User --> WH Ops engineer directly injecting one of these types of special files into our Flink cluster.

Do we need more detail than that?